ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx

```diff
@@ -74,6 +75,11 @@ export class WorkflowDrawer extends React.Component<WorkflowDrawerProps, Workflo
                         </div>
                     </div>
                 )}
+                <div className='workflow-drawer__section'>
+                    <div className='workflow-drawer__workflowFrom'>
+                        <WorkflowFrom namespace={wf.metadata.namespace} labels={wf.metadata.labels} />
+                    </div>
+                </div>
```

Comment:
![Screenshot 2020-08-05 at 10 10 51](https://user-images.githubusercontent.com/1142830/89443093-a8335980-d704-11ea-987d-51ab0820ebe8.png)
![Screenshot 2020-08-05 at 10 16 33](https://user-images.githubusercontent.com/1142830/89443169-c5682800-d704-11ea-8905-f9acb1be999f.png)

Reply:
I'm not sure I like this. A workflow can only be submitted from a single source (cwf, wftmpl, etc.) and this makes it seem like there could be more than one... almost as if it were labels or annotations. Can we make this a normal text field? `Submitted From: CronWorkflow omiscient-rhino` or something?

Reply:
Also, the icon is not entirely clean... I think we could benefit from spelling out `CronWorkflow`, etc.